### ISSUE TITLE
Organizer Session Rating: Rating some session results in unexpected error

### ISSUE THREAD

**Issue by mariobehling** [fossasia/open-event-frontend#6551]
Rating some session results in an unexpected error. It also shows sometimes two rows of stars then.

![Screenshot from 2021-01-30 20-42-50](https://user-images.githubusercontent.com/1583873/106366407-fcc63300-633b-11eb-810f-e55a18a3327f.png)

**Comment by maze-runnar** [comment]
on this.

**Comment by maze-runnar** [comment]
![Screenshot from 2021-02-05 16-15-04](https://user-images.githubusercontent.com/56407566/107024587-3c729c00-67ce-11eb-8258-ac9dbde61f37.png)

```sqlalchemy.exc.IntegrityError: (psycopg2.errors.NotNullViolation) null value in column "average_rating" violates not-null constraint
DETAIL:  Failing row contains (null, 1, first, null, , , , null, null, null, null, 1, null, null, null, null, null, null, 1, 1, pending, 2021-02-05 15:50:37.536562+05:30, 2021-02-05 15:50:37.449+05:30, null, f, 2021-02-05 15:55:05.620839+05:30, f, f, null, null).

[SQL: UPDATE sessions SET average_rating=(SELECT avg(feedback.rating) AS avg_1 
FROM feedback 
WHERE sessions.id = feedback.session_id) WHERE sessions.id IN (%(id_1)s)]
[parameters: {'id_1': 1}]
(Background on this error at: http://sqlalche.me/e/13/gkpj)

```
![Screenshot from 2021-02-05 16-19-40](https://user-images.githubusercontent.com/56407566/107024582-3aa8d880-67ce-11eb-9c7c-0f3b69ab4750.png)

**Comment by maze-runnar** [comment]
@iamareebjamal does it need to be fixed on server or frontend?

**Comment by iamareebjamal** [comment]
Server

**Comment by Creatoon** [comment]
@iamareebjamal yes sir i am also facing the same issue in pr #6537 , should i wait till this issue is resolved, thanks

**Comment by maze-runnar** [comment]
problem is here when destroying the feedback. 
if using this, no error is appearing
```
if (rating) {
      feedback.set('rating', rating);
    } else {
      feedback.set('rating', 1);
    }

```

**Comment by iamareebjamal** [comment]
@maze-runnar Please check if this fixes the issue https://github.com/fossasia/open-event-server/pull/7712

**Comment by maze-runnar** [comment]
ok

**Comment by maze-runnar** [comment]
> @maze-runnar Please check if this fixes the issue [fossasia/open-event-server#7712](https://github.com/fossasia/open-event-server/pull/7712)

yes it fixed the issue🔥
